### ISSUE TITLE
3.6.0 doesn't record the file list correctly

### ISSUE THREAD

**Issue by yurivict** [RaRe-Technologies/gensim#2237]
The plist that it records using the ```--record``` option is wrong:
```
===> Checking for items in pkg-plist which are not in STAGEDIR
Error: Missing: %%PYTHON_SITELIBDIR%%/gensim/models/doc2vec_corpusfile.so
Error: Missing: %%PYTHON_SITELIBDIR%%/gensim/models/fasttext_corpusfile.so
Error: Missing: %%PYTHON_SITELIBDIR%%/gensim/models/word2vec_corpusfile.so
```

Found in the FreeBSD port.

**Comment by yurivict** [comment]
Same as https://github.com/RaRe-Technologies/gensim/issues/2198
#2198 was closed but the issue came back now.

**Comment by menshikh-iv** [comment]
@yurivict hello, are you get install gensim from PyPI? The issue really already fixed in #2194, this is just not released yet.

**Comment by yurivict** [comment]
Ok, thanks, I will wait for the release then.
